bin/myresources.py

```diff
@@ -32,11 +32,11 @@
 
 from __future__ import division, print_function
 from argparse import ArgumentParser, RawDescriptionHelpFormatter
+import os
 import sys
+import tempfile
 import xml.etree.cElementTree as ET
 
-from vsc.utils.run import run
```

Comment:
can't you use `asyncloop` from `vsc.utils.run` instead?

Reply:
that works too, thanks.
I'd tried `noworries` and `async_run`, which all failed.